### ISSUE TITLE
make cardType usable in faceTemplates

### ISSUE THREAD

**Issue by ArnoldSmith86** [ArnoldSmith86/virtualtabletop#1129]
I do stuff like this all the time because I can't put the key onto the card:

```JSON
  "cardTypes": {
    "Boot": {
      "main": "Boot",
      "forbidden1": "Wasser",
      "forbidden2": "Fahrzeug",
      "forbidden3": "Schiff",
      "forbidden4": "Segel",
      "forbidden5": "Yacht"
    }
  }
```

**Comment by ArnoldSmith86** [comment]
Oh. My. God. I just wanted to implement this and realized that you can totally use `cardType` inside a `faceTemplates -> objects -> dynamicProperties` because it's a property of the card.

**Comment by 96LawDawg** [comment]
I’m not sure I understand the example. Could you give a use case or better explain the example. Seems like we should document this in the wiki. I can do that if you explain it to me.

**Comment by ArnoldSmith86** [comment]
I usually handcraft my decks and their cardTypes so I use useful names for the cardType (in this example `Boat`). And because it identifies the card, it quite often is also written on the card. So to use `Boat` inside the `faceTemplates`, I always added another property to the cardType containing the same string (`Boat`) so I could use that in the templates.

It might not be news to you at all that `cardType` works just fine. I just never realized it.

**Comment by 96LawDawg** [comment]
That makes sense.  I don't see a clean way to add that in to the current wiki without breaking up the flow of the stuff that is already there.  I tested out your idea in Flora where I handcrafted my deck like you said.  It worked fine (except that I have Rose1, Rose2, Rose3, ... Rose8, so I still need a custom property of "flower" because I can't use the actual cardType name or I also get the number.  Unless you know of a good way to create 8 cards that are all the same except for the numbers 1-8 on them??
